### ISSUE TITLE
fltering by more than one tag

### ISSUE THREAD

**Issue by codigo-osv** [jazzband/tablib#251]
Hello, when I read about this method here http://docs.python-tablib.org/en/latest/tutorial/#filtering-datasets-with-tags I got the impression that I could pass a list of tags and it would filter by all of them ("students.filter(['male'])" is in the example), but this only seems to work by calling filter once for each tag. 

Regards.

**Comment by xdanielsb** [comment]
Hi @codigo-osv,

please find some examples of the correct usage,
```python

students = tb.Dataset()
students.headers = ['first', 'last']
students.rpush(['Kenneth', 'Reitz'], tags=['male', 'technical'])
students.rpush(['Daniel', 'Dupont'], tags=['male', 'creative'])
students.rpush(['Bessie', 'Monke'], tags=['female', 'creative'])

```


### Filter ( and  )
```sh
>>> students.filter(['female']).filter(['creative'])
{first: Bessie, Last: Monke}
```
### Filter ( or )
```sh
students.filter(['female', 'creative'])
{first: Daniel, Last: Dupont}
{first: Bessie, Last: Monke}
```

regards

**Comment by xdanielsb** [comment]
@claudep  I did not realize of the message before :v , now the doc is updated regarding this issue.

**Comment by claudep** [comment]
Fixed by d94420d96805f0
